e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -1116,3 +1107,27 @@ function assertScrollBarExists() {
     cy.window().its("innerWidth").should("be.gte", bodyWidth);
   });
 }
+
+function addParameterToDashboard() {
```

Comment:
If the purpose of the test is not to test the process of adding and mapping a parameter, I'd move all these actions to API calls instead. Since that saves time.

You can create a parameter when creating the dashboard by e.g.
```ts
      cy.createDashboard({
        parameters: [{
        name: "Text",
        slug: "text",
        id: "94f9e513",
        type: "string/=",
        sectionId: "string",
        default: ["Example Name"],
      }]
```

and map that parameter to a card by passing this property to a dashcard
```ts
parameter_mappings: [
  { parameter_id: "94f9e513", target: ["text-tag", "Name"] },
],
```